src/pages/Login/Login.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file Login.js
+ * Exports a React component that renders EditVR's login.
+ */
+
+import React from 'react';
+import { Typography } from '@material-ui/core';
+
+import { LoginForm } from '../../components';
+import { ThinLayout } from '../../layouts';
+
+const Login = () => (
+  <ThinLayout>
+    <Typography variant="headline">Log into EditVR</Typography>
+    <LoginForm />
```

Comment:
Login page, which is rendered for the `/login` route, renders a title and the login form.